lib/tasks/dev.rake

```diff
@@ -193,6 +195,14 @@ namespace :openfoodnetwork do
       CreateOrderCycle.new(enterprise2, variants).call
 
       EnterpriseRole.create!(user: Spree::User.first, enterprise: enterprise2)
+
+      confirm_user(spree_user)
```

Comment:
The Devise `Confirmable` module provides a [confirm!](https://github.com/plataformatec/devise/blob/beeb48c404ae8b4c3c34f6d0a2325ca2bcc9298e/lib/devise/models/confirmable.rb#L58) method, so perhaps you could just do`spree_user.confirm!`.